parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -316,6 +316,5 @@ coreos:
 runcmd:
 - set -x
 - . /opt/azure/containers/provision_source.sh
-- timeout 10 apt-mark hold walinuxagent{{GetKubernetesAgentPreprovisionYaml .}}
```

Comment:
Remove `apt-mark hold|unhold walinuxagent` from cloud-init implementation as all cluster bootstrapping runtime implementation is in CSE.